### PR TITLE
[RFR][MINOR] Feature/UpgradeStuff

### PR DESCRIPTION
**Pull request by chrisamoore**

### CODE REVIEW

**Comment by chrisamoore** [comment]
@keithajackson would you take a look?

**Comment by chrisamoore** [comment]
@nVitius

File: .zsh_git [hunk 1]
```diff
@@ -18,7 +18,7 @@ alias gfo='git fetch origin'
 alias grv='git remote -v'
 alias gra='git remote add'
 alias grm='git remote rm'
-alias gst='git status'
+alias gst=' find . -regex '.*-2016-.*' -delete && git status'
```
Comment (nVitius) [inline .zsh_git]: what?

File: .vim/.netrwhist [hunk 2]
```diff
@@ -1,4 +1,6 @@
 let g:netrw_dirhistmax  =10
-let g:netrw_dirhist_cnt =2
+let g:netrw_dirhist_cnt =4
 let g:netrw_dirhist_1='/Users/cmoore/Code/slc-docker'
 let g:netrw_dirhist_2='/Users/chrismoore'
+let g:netrw_dirhist_3='/Users/chrismoore/.oh-my-zsh'
+let g:netrw_dirhist_4='/Users/chrismoore/.oh-my-zsh/themes'
```
Comment (nVitius) [inline .vim/.netrwhist]: you can use `$HOME` in these too.

File: .zsh_profile [hunk 3]
```diff
@@ -57,8 +57,8 @@ alias fs="stat -f \"%z bytes\""
 
 # Empty the Trash on all mounted volumes and the main HDD
 alias emptytrash="sudo rm -rfv /Volumes/*/.Trashes; rm -rfv ~/.Trash"
-alias kmongo='mongo admin --eval "db.shutdownServer()" > /dev/null'
+# alias kmongo='mongo admin --eval "db.shutdownServer()" > /dev/null'
 alias admongo='cd ~/Code/tools/adminMongo && pm2 start app.js -i 1 -n mongo-admin && cd -'
```
Comment (nVitius) [inline .zsh_profile:61]: maybe use a `$CODE` alias?

**Pull request merged**